js/renovation/ui/editors/check_box/check_box.tsx

```diff
@@ -100,14 +100,16 @@ export class CheckBoxProps extends EditorProps {
 
   @OneWay() iconSize?: number | string;
 
+  @OneWay() indeterminateStateEnabled = false;
```

Comment:
We decided name this option as `allowIndeterminateStateByClick`

Reply:
https://github.com/DevExpress/DevExtreme/pull/22616/commits/27d04ce3306d5b9db1919f51aa1cb3e63972f829